tests/test_cmdline.py

```diff
@@ -68,7 +68,7 @@ def test_dump(script_runner):
     ret = script_runner.run('tnefparse', '-d', 'tests/examples/two-files.tnef')
     assert ret.success
     dump = json.loads(ret.stdout)
-    assert sorted(list(dump.keys())) == ['attachments', 'attributes', 'extended_attributes']
+    assert sorted(dump) == ['attachments', 'attributes', 'extended_attributes']
```

Comment:
Wow, I learned something new.

Following the Zen of Python, I'd still personally prefer:
```
assert sorted(dump.keys())
```
even when it is not necessary.